src/pages/workspace/categories/CategoryRequireReceiptsOverPage.tsx

```diff
@@ -0,0 +1,119 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import * as Category from '@userActions/Policy/Category';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type EditCategoryPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORY_REQUIRE_RECEIPTS_OVER>;
+
+function getInitiallyFocusedOptionKey(isAlwaysSelected: boolean, isNeverSelected: boolean): ValueOf<typeof CONST.POLICY.REQUIRE_RECEIPTS_OVER_OPTIONS> {
+    if (isAlwaysSelected) {
+        return CONST.POLICY.REQUIRE_RECEIPTS_OVER_OPTIONS.ALWAYS;
+    }
+
+    if (isNeverSelected) {
+        return CONST.POLICY.REQUIRE_RECEIPTS_OVER_OPTIONS.NEVER;
+    }
+
+    return CONST.POLICY.REQUIRE_RECEIPTS_OVER_OPTIONS.DEFAULT;
+}
+
+function performAction(policyID: string, categoryName: string, value: number | null) {
+    if (typeof value === 'number') {
+        Category.setPolicyCategoryReceiptsRequired(policyID, categoryName, value);
+        return;
+    }
+
+    Category.removePolicyCategoryReceiptsRequired(policyID, categoryName);
+}
```

Comment:
it's used only in one place, maybe it should just be defined inside of the page as a complete handler for onSelectRow?